kombu/transport/redis.py

```diff
@@ -485,6 +486,11 @@ class Channel(virtual.Channel):
     #: Disable for backwards compatibility with Kombu 3.x.
     fanout_patterns = True
 
+    #: The global key prefix will be prepended to all keys used
+    #: by Kombu, which can be useful when a redis database is shared
+    #: by different users. By default, no prefix is prepended.
+    global_keyprefix = ''
```

Comment:
this needs to be documened

Reply:
@auvipy Should I regenerate the documentation somehow or is this something I'd need to do somewhere else?